include/sparrow/buffer/dynamic_bitset/concepts.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright 2024 Man Group Operations Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <ranges>
+
+namespace sparrow
+{
+    template <class T>
+    concept random_access_range = std::ranges::random_access_range<T>;
```

Comment:
Why not `using std::ranges::random_access_ranges` where we need it instead?

Reply:
I removed the concept 👍